changelog/18.0/18.0.0/summary.md

```diff
@@ -238,3 +240,20 @@ removing Vitess support.
 #### <a id="new-durability-policies"/>New Durability Policies
 
 2 new inbuilt durability policies have been added to Vitess in this release namely `semi_sync_with_rdonly_ack` and `cross_cell_with_rdonly_ack`. These policies are exactly like `semi_sync` and `cross_cell` respectively, and differ just in the part where the rdonly tablets can also send semi-sync ACKs. 
+
+### <a id="foreign-keys"/>Experimental Foreign Key Support
+
+A new field `foreignKeyMode` has been added to the Vschema. This field can be provided for each keyspace. The Vtgate flag `--foreign_key_mode` has been deprecated in favour of this field.
```

Comment:
For me being not fully aware of how foreign keys work, it is unclear if I can go from one mode to the other for a given keyspace, or what happens if I start with (for instance) `FK_MANAGED` and decide to move to `FK_UNMANAGED`.

Reply:
Just change the Vschema. It is a vschema property.